Sources/SwiftCliCore/Errors.swift

```diff
@@ -3,8 +3,17 @@ import Foundation
 enum InputError: Error, Equatable {
     case emptyRow
     case invalidStitch(invalidStitch: String, rowLocation: Int? = nil)
-    case invalidRowWidth(invalidRowNumber: Int, expectedStitchCount: Int, actualCount: Int)
-    case invalidStitchNumber(rowNumber: Int? = nil, invalidStitch: String, validStitchType: String, invalidStitchNumber: String)
+    case invalidRowWidth(
+        invalidRowNumber: Int,
+        expectedStitchCount: Int,
+        actualCount: Int
+    )
+    case invalidStitchNumber(
+        rowNumber: Int? = nil,
+        invalidStitch: String,
+        validStitchType: String,
+        invalidStitchNumber: String
```

Comment:
Does it make more sense to store this as `Int` to be consistent with how you store numbers in other error cases?

Reply:
See above - have fixed this on `MultipleErrorArray` branch!